ec2utils/ec2uploadimg/ec2uploadimg

```diff
@@ -514,6 +520,7 @@ try:
                           root_volume_size=root_volume_size,
                           running_id=running_id,
                           secret_key=secret_key,
+                          session_token=args.sessionToken,
```

Comment:
Breaks alpha order should be "...secret_key,security_group_ids,session_token...".